modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/forms/input/InputLocalized.tsx

```diff
@@ -24,6 +24,7 @@ interface InputLocalizedProps {
 	onSelectedLocaleChange?: (locale: Liferay.Language.Locale) => void;
 	placeholder?: string;
 	required?: boolean;
+	resultFormatter?: (val: string) => React.ReactNode;
```

Comment:
A while back we had an initiative where we pruned the codebase of abbreviated props such as this `val` here. So if you could rename it to `value` that would be good.

We have an eslint rule for it, but `val` might not be on the blacklist. https://github.com/liferay/liferay-frontend-projects/blob/master/projects/eslint-plugin/rules/general/docs/rules/no-abbreviations.md